### ISSUE TITLE
alt text on images in blog posts

### ISSUE THREAD

**Issue by boydnorwood** [nozzle/nozzle.io#67]
When I upload an image for the first time, whatever I use as the Title of that image becomes the alt text of the image when inserted into a post.

Then if I go back to that image in the media library and change the title a little to optimize the alt text better, when I hit save and even if I republish the blog post, the alt text stays the same as what it was before even though the title of the image is now altered.

Can you add an alt text field in the media library for each image so we can easily update and test different alt texts?

**Comment by jtribe32** [comment]
Unfortunately Contentful does not allow you to edit the fields in the media library. I could however, make a new content model like author or blog post that includes an alt text field that then could be referenced for the featured image section of the blog post. 

The problem is that still doesn't work in the body field. To change the alt text in the body you can edit the reference of the markdown it gives you. It's in the format of ![alt text](url). Would you like me to make a new content model for the featured image?
